arcgis-ios-sdk-samples/Content Display Logic/Controllers/ContentTableViewController.swift

```diff
@@ -28,29 +28,51 @@ class ContentTableViewController: UITableViewController {
     var allSamples = [Sample](){
         didSet{
             displayedSamples = allSamples
-            searchEngine = SampleSearchEngine(samples: allSamples)
+            if !containsSearchResults{
+                searchEngine = SampleSearchEngine(samples: allSamples)
+            }
         }
     }
-    private var expandedRowIndex:Int = -1
-    
-    var containsSearchResults = false
-    private var bundleResourceRequest:NSBundleResourceRequest!
-    private var downloadProgressView:DownloadProgressView!
+    private var expandedRowIndex: Int = -1
     
-    var searchEngine: SampleSearchEngine?
-    
-    // strong reference needed for iOS 10
-    var filterSearchController:UISearchController?
+    private var bundleResourceRequest: NSBundleResourceRequest!
```

Comment:
I know this isn't related to the changes you made but it would be nice if these to variables were not implicitly unwrapped optionals.

Reply:
Fixed.